src/main/java/org/coffeemine/app/spring/data/Task.java

```diff
@@ -12,6 +12,8 @@
     @NotNull
     private String name = "";
     @NotNull
+    private String assignSprint = "";
```

Comment:
even if we want to continue using this you should at least store the sprint id integer and not string

Reply:
We shouldn't backtrack sprints from tasks